specification/compute/resource-manager/Microsoft.Compute/stable/2018-04-01/compute.json

```diff
@@ -4363,6 +4631,89 @@
       ],
       "description": "The List Availability Set operation response."
     },
+    "ProximityPlacementGroupProperties": {
+      "properties": {
+        "proximityPlacementGroupType": {
+          "type": "string",
+          "description": "Specifies the type of the proximity placement group. <br><br> Possible values are: <br><br> **Standard** <br><br> **Ultra**",
+          "enum": [
+            "Standard",
+            "Ultra"
+          ],
+          "x-ms-enum": {
+            "name": "ProximityPlacementGroupType",
+            "modelAsString": true
+          }
+        },
+        "virtualMachines": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResource"
+          },
+          "description": "A list of references to all virtual machines in the proximity placement group."
+        },
+        "virtualMachineScaleSets": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResource"
+          },
+          "description": "A list of references to all virtual machine scale sets in the proximity placement group."
+        },
+        "availabilitySets": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResource"
+          },
+          "description": "A list of references to all availability sets in the proximity placement group."
+        }
+      },
+      "description": "Describes the properties of a Proximity Placement Group."
+    },
+    "ProximityPlacementGroup": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ProximityPlacementGroupProperties",
+          "description": "Describes the properties of a Proximity Placement Group."
+        },
```

Comment:
remove this comma, this causes invalid Json

Reply:
As @lmazuel pointed, fix this in 2018-10-01 and 2018-06-01 as well

Reply:
Fixed